Sources/ContainerBuilder/ContainerBuilder.swift

```diff
@@ -75,35 +100,37 @@ public func buildContainer(
     let repositoriesData = try JSONEncoder().encode(repositories)
     let repositoriesPath = tempDir.appendingPathComponent("repositories")
     try repositoriesData.write(to: repositoriesPath)
-    
+
     // Create final container image tarball
     let imageDir = tempDir.appendingPathComponent("image")
     try FileManager.default.createDirectory(at: imageDir, withIntermediateDirectories: true)
-    
-    // Copy layer and config to image directory
-    let imageLayerPath = imageDir.appendingPathComponent("\(layerSHA).tar")
-    try FileManager.default.copyItem(at: layerTarPath, to: imageLayerPath)
-    
+
+    // Copy layers and config to image directory
+    for (layerSHA, layerPath) in zip(layerSHAs, layerPaths) {
+        let imageLayerPath = imageDir.appendingPathComponent("\(layerSHA).tar")
+        try FileManager.default.copyItem(at: layerPath, to: imageLayerPath)
```

Comment:
```suggestion
    for (layerSHA, originPath) in zip(layerSHAs, layerPaths) {
        let destinationPath = imageDir.appendingPathComponent("\(layerSHA).tar")
        try FileManager.default.copyItem(at: originPath, to: destinationPath)
```

This could be a bit more clearly named